tools/train.py

```diff
@@ -264,7 +264,8 @@ def main():
                 drop_last=getattr(cfg.data, 'drop_last', False),
                 reuse_worker_cache=cfg.data.get('reuse_worker_cache', False),
                 persistent_workers=cfg.data.get('persistent_workers', False),
-                collate_hooks=cfg.data.get('train_collate_hooks', []))
+                collate_hooks=cfg.data.get('train_collate_hooks', []),
+                repeated_aug=cfg.data.get('repeated_aug', False))
```

Comment:
Literally i do not understand the meaning of the `repeated_aug`, can named it with a more understandable name?

Reply:
I rename repeated_aug to use_repeated_augment_sampler. If set True, it means using RASampler.